src/backend/web/graphql/index.js

```diff
@@ -113,37 +113,56 @@ module.exports.resolvers = {
      * @return Array of 'perPage' number of Post objects
      */
     getPosts: async (parent, { filter, page, perPage }) => {
+      let postIds;
+      const numOfPosts = await getPostsCount();
```

Comment:
Do this later: depending on how we filter, it will be different (all posts vs. date range)